tests/unit/test_guess_packager.py

```diff
@@ -0,0 +1,124 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import subprocess
+from collections.abc import Iterable
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+import rpm
+
+from specfile.changelog import guess_packager
+
+
+@pytest.fixture
+def clean_guess_packager(
+    tmp_path: Path, monkeypatch: pytest.MonkeyPatch
+) -> Iterable[None]:
+    """
+    Ensure a clean environment
+    """
+    # For $RPM_PACKAGER
+    monkeypatch.delenv("RPM_PACKAGER", False)
+    # Make sure git doesn't read existing config
+    monkeypatch.setenv("HOME", "/dev/null")
+    monkeypatch.delenv("XDG_CONFIG_HOME", False)
+    monkeypatch.chdir(tmp_path)
+    # For %packager
+    old = rpm.expandMacro("%packager")
```

Comment:
I don't think this is worth preserving, all macros are reset anyway every time `Specfile` is instantiated and on plenty other occasions.

Reply:
https://github.com/packit/specfile/pull/220/commits/e4c5bd829e37ea2eed680649e3a0049b3b370a3e